src/main.cpp

```diff
@@ -3246,7 +3246,7 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
                 } else if (out.scriptPubKey.IsPayToPublicKey() || out.scriptPubKey.IsColdStaking()) {
                     uint160 hashBytes;
                     int type = 0;
-		    CTxDestination destination;
```

Comment:
Maybe we should add `.editorconfig` to the project?

Reply:
So we can standardize the white space and some other basic configs

Reply:
+1